packages/material-ui-styles/src/makeStyles.js

```diff
@@ -199,55 +191,52 @@ function makeStyles(stylesOrCreator, options = {}) {
       ...stylesOptions2,
     };
 
-    const { current: state } = React.useRef({});
+    const { current: instance } = React.useRef({
+      // previous: null,
+      // current: null,
```

Comment:
There are enough useTimeTravel hooks in the wild. No need to reinvent.

Reply:
Do you have an example?